source/Ship.cpp

```diff
@@ -2300,6 +2300,18 @@ void Ship::DoGeneration()
 				if(fuelRemaining > 0.)
 					DoRepair(ship.fuel, fuelRemaining, ship.attributes.Get("fuel capacity"));
 			}
+
+			// Carried ships can steal energy from their parent's batteries,
```

Comment:
Why this got reverted?
```suggestion
			// Carried ships can recharge energy from their parent's batteries,
```

Reply:
No idea. Probably a mishandled merge conflict or something.